src/auction/utils.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable camelcase, compat/compat */
+
+/*
+ * @function
+ * @name getEligible
+ * @description filter all buyers by their owner field to ensure they're eligible to bid
+ * @author Newton <cnewton@magnite.com>
+ * @param {array<Object>} buyers - an array of objects containing interest group buyers
+ * @param {array<String>} eligible - a list of eligible owners to check against
+ * @return {Array<Object> | null} an array of objects; null if none found;
+ */
+export const getEligible = (groups, eligibility) => {
+	if (eligibility === '*') {
+		return groups;
+	}
+
+	const eligible = groups.filter(({ owner }) => eligibility.includes(owner));
+	if (eligible.length) {
+		return eligible;
+	}
+
+	return null;
+};
+
+/*
+ * @function
+ * @name getBids
+ * @description given a set of bidders, grab their bid
+ * @author Newton <cnewton@magnite.com>
+ * @param {array<Object>} bidders - a list of eligible owners to bid
```

Comment:
```suggestion
 * @param {array<Object>} interestGroups - a list of eligible interest groups to generate bids from
```

Reply:
at this time, going to go with bidders; if the general consensus is to change it, will do so at that time